lib/shopify_transporter/shopify/customer.rb

```diff
@@ -57,12 +57,36 @@ def to_csv
       ).freeze
 
       def top_level_row_values
-        base_hash.merge(record_hash.slice(*TOP_LEVEL_ATTRIBUTES)).values
+        return top_level_attributes.values unless able_to_merge_address?
+
+        top_level_attributes.merge(default_address.compact).values_at(*Customer.columns)
+      end
+
+      def top_level_attributes
+        base_hash.merge(record_hash.slice(*TOP_LEVEL_ATTRIBUTES))
+      end
+
+      def default_address
+        return {} unless at_least_one_address?
+
+        record_hash['addresses'][0].slice(*ADDRESS_ATTRIBUTES)
+      end
+
+      def at_least_one_address?
+        record_hash['addresses'].present? && record_hash['addresses'][0].present?
+      end
+
+      def able_to_merge_address?
```

Comment:
I suggest being more specific with the method name: `able_to_merge_default_address?`